lib/src/app/app_flavors.dart

```diff
@@ -0,0 +1 @@
+enum Flavor { dev, prod }
```

Comment:
może Environment? ;)

Reply:
imo nie ma większego znaczenia, po słówku Flavor też wiadomo o co chodzi i tak jest to nazywane w dokumentacji fiuterowej, ale jak bardziej Ci się podoba environment to może być environment ;)